packages/components/organisms/f-content-cards/src/components/cardTemplates/StampCard1.vue

```diff
@@ -80,6 +80,13 @@
 import parseISO from 'date-fns/parseISO';
 import format from 'date-fns/format';
 import lightFormat from 'date-fns/lightFormat';
+import {
+    da, enAU, enNZ, enGB, es, it, nb
+} from 'date-fns/locale';
+import EmptyStamp15 from './images/stamp-empty-15.svg';
```

Comment:
Are there plans to move these into the icon package potentially?

Reply:
yes but atm they are in here until I move them out